draft-ietf-quic-tls.md

```diff
@@ -504,6 +504,9 @@ client could interleave ACK frames that are protected with Handshake keys with
 0-RTT data and the server needs to process those acknowledgments in order to
 detect lost Handshake packets.
 
+A TLS implementation MUST NOT provide a 1-RTT decrypt secret to QUIC until it
+has received a Finished message from the peer.
```

Comment:
```suggestion
the TLS handshake is complete.
```

Receiving a Finished message is not sufficient, as you have to verify it.  Also, at least in our stack, verifying the Finished is not the only condition we hold handshake completion on.  We also wait until the certificate is marked as "OK", which is an asynchronous process and so might happen after validating the Finished.